### PR TITLE
Text to LANs -#6 Flexpanel

### PR DESCRIPTION
**Pull request by MikeyGMT**
Text to LANs -#6 Flexpanel

### CODE REVIEW

**Comment by MikeyGMT** [comment]
Testing 

image

**Comment by MikeyGMT** [comment]
I could not manage to screenshot the Toggle Sidebar Text.

File: e107_admin/includes/layouts/flexpanel_wider_sidebar.php [hunk 1]
```diff
@@ -14,7 +20,7 @@
 	</div>
 	<div class="col-md-9 col-lg-9 admin-right-panel">
 		<div class="sidebar-toggle">
-			<a href="#" title="Toggle Sidebar" data-toggle-sidebar="true">&nbsp;</a>
+			<a href="#" title='.ADLAN_185.' data-toggle-sidebar="true">&nbsp;</a>
```
Comment (Moc) [inline e107_admin/includes/layouts/flexpanel_wider_sidebar.php]: Shouldn't this be?:

```php
 title="'.ADLAN_185.'"
```

**Comment by MikeyGMT** [comment]
Well spotted I'll correct tomorrow

**Comment by MikeyGMT** [comment]
Fixed in last commit

**Pull request merged**